aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/SyncProcessorTest.java

```diff
@@ -400,6 +496,48 @@ public void userProvidedErrorCallbackInvokedOnFailure() {
         assertEquals(1, errorHandlerCallCount);
     }
 
+    private void assertOperationTypeCounts(ModelSyncedEvent event,
+                                           int expectedAdd,
+                                           int expectedUpdate,
+                                           int expectedDelete) {
```

Comment:
When you call this, it is hard to remember which int is which. While potentially overkill, one solution is like:
```java
assertOperationTypeCounts(event, TypeCounts.builder().added(5).updated(1).deleted(0).build());
```
With a little inner data class to store those vals.

Or hey, I mean, it's just a test class. Life doesn't always need to be complicated, we could even just have:
```
class TypeCounts {
    int added;
    int updated;
    int deleted;
}
```
And then prepare them:
```java
TypeCounts counts = new TypeCounts();
counts.added = 4;
counts.updated = 9;
counts.deleted = 1;
```

Reply:
Oh duh, or wait -- update. Can you just reuse the `ModelSyncMetrics` as a pojo to store these immediate values?

Reply:
Oh yeah...I don't even have to have a separate method. I just created "expected" objects and use `assertEquals`.